src/main/java/core/basesyntax/temp/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.temp;
+
+public interface AreaCalculator {
+    public abstract void getArea();
```

Comment:
```suggestion
    Double getArea();
```

всі методи без реалізації в інтерфейсі public abstract  за замовчуванням